reference_data/centos7.yml

```diff
@@ -0,0 +1,116 @@
+---
+- name: CentOS7
+  distr_type: rhel
+  distr_version: '7'
+  #? is this email correct ?#
+  contact_mail: packager@centos.org 
+  copyright: Copyright {{ year }} CentOS
+  test_dist_name: centos7
+  modularity:
+    #? I cant find CentOS alternative for modified_packages.yml ?#
+    modified_packages_url: 'https://git.almalinux.org/almalinux/modified_packages/raw/branch/a8/modified_packages.yml'
+    packages_git: 'https://git.centos.org/projects/rpms/%2A'
+    #? what tag to use here ?#
+    git_tag_prefix:
+      modified: a8
+      non_modified: c8
+    versions:
```

Comment:
These are module-related versions, they are not needed for CentOS 7. But we need to check if this data is required to store the entry in the database or during the build creation process